pkg/db/topology.go

```diff
@@ -15,6 +16,16 @@ import (
 	"gorm.io/gorm/clause"
 )
 
+func GetTeamsOfComponents(ctx context.Context, componentIDs []string) ([]string, error) {
+	var teams []string
+	err := Gorm.WithContext(ctx).Table("team_components").Select("DISTINCT teams.name").Joins("LEFT JOIN teams ON teams.id = team_components.team_id").Where("component_id IN (?)", componentIDs).Find(&teams).Error
+	if err != nil {
+		return teams, err
+	}
+
+	return teams, nil
```

Comment:
```suggestion
	return teams, err
```

No need for the if check